### PR TITLE
GH actions cleanup

### PR DESCRIPTION
**Pull request by heeckhau**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -52,6 +52,27 @@ at your option.
...
+> ```
+
+If you run into this error:
+```
+Could not find directory of OpenSSL installation, and this `-sys` crate cannot
+  proceed without this knowledge. If OpenSSL is installed and this crate had
+  trouble finding it,  you can set the `OPENSSL_DIR` environment variable for the
+  compilation process.
+```
+Make sure you have the development packages of OpenSSL installed (`libssl-dev` on Ubuntu or `openssl-devel` on Fedora).
```
Comment (yuroitaki) [inline README.md:72]: What about MacOS users?

File: README.md [hunk 2]
```diff
@@ -52,6 +52,27 @@ at your option.
...
+
+If you want to contribute; Thanks a lot! please check <CONTRIBUTING.md> first.
```
Comment (yuroitaki) [inline README.md:74]:
```suggestion
If you want to contribute, thanks a lot! Please check [CONTRIBUTING.md](CONTRIBUTING.md) first.
```

File: crates/wasm/build.sh [hunk 3]
```diff
@@ -0,0 +1,3 @@
+RUSTFLAGS='-C target-feature=+atomics,+bulk-memory,+mutable-globals' \
+    rustup run nightly \
+    wasm-pack build --target web . -- -Zbuild-std=panic_abort,std
```
Comment (yuroitaki) [inline crates/wasm/build.sh:3]: The command in https://github.com/tlsnotary/tlsn/blob/dev/crates/wasm/README.md doesn't have `--`, should we stick to one?

**Review by yuroitaki — COMMENTED** [review]
Looks good~ just a few comments

File: README.md (hunk 2 above)
Comment (heeckhau) [reply README.md:74]: I'll drop this line. There is Contribution section below it

File: README.md (hunk 1 above)
Comment (yuroitaki) [reply README.md:72]: P/S: Doesn't seem to affect MacOS users

**Review by yuroitaki — APPROVED** [review]

**Review by themighty1 — APPROVED** [review]

**Pull request merged**